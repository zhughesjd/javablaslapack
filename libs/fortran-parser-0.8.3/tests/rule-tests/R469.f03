

! Test of array_value_list.
x = (/3.1, 4.1, 5.9265/)
end
