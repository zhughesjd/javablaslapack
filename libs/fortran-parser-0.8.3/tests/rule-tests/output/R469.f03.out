1101:begin
613 x false false
612 1
603 false
601
469:list-begin
417 3 .1 null
417 4 .1 null
417 5 .9265 null
469:list 3
465
734
1103
1101 false true false
